.github/workflows/test_pull_requests.yml

```diff
@@ -88,15 +88,23 @@ jobs:
             platform: ubuntu-latest
             backend: pyqt5
             pydantic: "_pydantic_1"
+            coverage: no_cov
           - python: 3.11
             platform: ubuntu-latest
             backend: pyqt6
             pydantic: ""
+            coverage: no_cov
+          - python: 3.8
+            platform: ubuntu-20.04
+            backend: pyqt5
+            MIN_REQ: 1
+            coverage: cov
     with:
       python_version: ${{ matrix.python }}
       platform: ${{ matrix.platform }}
       qt_backend: ${{ matrix.backend }}
-      coverage: no_cov
+      coverage: ${{ matrix.coverage }}
+      min_req: ${{ matrix.MIN_REQ }}
```

Comment:
This is to simplify copy and paste problematic job to initial section and speedup CI.